RespectTransWomenPublic.py

```diff
@@ -28,22 +46,42 @@
         print("Comment is by myself")
         continue
 
-    if text.lower == "Shemale" or "Ladyboy" or "tranny" or "trannie" or "she-man" or "transvestite" or "chick with dick" or "chicks with dicks" or "dickgirl" or "men with tits" or "HeShe" or "He-She": #IDK if I implemented non case sensitivity right. its 3 am... ORIGINAL COMMENT HERE: check to see the comment is "!lmgtfy". use if text.lower() == "!lmgtfy".lower() to be non-case sensitive, use if "!lmgtfy" in text if you want the comment to be anywhere
-        if comment.id in f.read(): #if the comment is already in the file, bot has replied to it
+    caught_words = []
+    for i in trigger_words:
+        if (i in text) and not (i in caught_words): #Check if the comment contains a word we're looking for. not (i in caught_words) is technically redundant but prevents the word being mentioned by the bot multiple times if it's repeated in trigger_words
+            caught_words.append(i)
+    message = "Please do not use "
+    caught_len = len(caught_words)
+    caught_iter = 1
+    if caught_len > 2:
+        for i in caught_words:
+            if caught_len - caught_iter > 0: #Format the message in the form of "Please do not use a, b, or c. It is..."
+                message += i + ", "
+            else:
+                message += "or " + i
+            caught_iter += 1
+    elif caught_len == 2:
+        for i in caught_words:
+            if caught_len - caught_iter > 0: #Format the message in the form of "Please do not use a or b. It is..."
+                message += i
+            else:
+                message += " or " + i
+            caught_iter += 1
+    else:
+        for i in caught_words:

```

Comment:
This definitely should just be accessed directly but I'm too tired to do it properly atm